cmd/gaia/app/genesis.go

```diff
@@ -86,18 +86,21 @@ type GaiaGenTx struct {
 }
 
 // Generate a gaia genesis transaction with flags
-func GaiaAppGenTx(cdc *wire.Codec, pk crypto.PubKey) (
+func GaiaAppGenTx(cdc *wire.Codec, pk crypto.PubKey, config *gaiacfg.Config) (
```

Comment:
A design pattern we're trying to follow in the SDk: let's not make `config` a pointer here, if we want to modify config, let's return a new copy of it.